autocomplete_test.go

```diff
@@ -0,0 +1,138 @@
+package input_autocomplete
+
+import (
+	"errors"
+	"testing"
+)
+
+type DirListerCustomMock struct {
+	listContentMock func(path string) ([]string, error)
+}
+
+func (d DirListerCustomMock) ListContent(path string) ([]string, error) {
+	return d.listContentMock(path)
+}
+
+func Test_autocomplete_unixAutocomplete(t *testing.T) {
```

Comment:
Could you check the OS the user is running the test? Because if the user is running windows the test will fail. You can do something similar to the `input_test.go` file at line 141.

Reply:
Sure!